### PR TITLE
Update Smokeping.pm

### PR DESCRIPTION
**Pull request by fanyeren**
add support for smtp authentication

### CODE REVIEW

File: lib/Smokeping.pm [hunk 1]
```diff
@@ -2551,7 +2553,7 @@ General configuration values valid for the whole SmokePing setup.
 DOC
          _vars =>
          [ qw(owner imgcache imgurl datadir dyndir pagedir piddir sendmail offset
-              smokemail cgiurl mailhost snpphost contact display_name
+              smokemail cgiurl mailhost mailuser mailpass snpphost contact display_name
```
Comment (oetiker) [inline lib/Smokeping.pm]: please add _doc entries for the new variables ...

Comment (fanyeren) [inline lib/Smokeping.pm]: done, @oetiker

Comment (oetiker) [inline lib/Smokeping.pm]: geat

**Comment by oetiker** [comment]
thanks

**Pull request merged**